README.md

```diff
@@ -1,24 +1,95 @@
-# CAR FOR YOU Example
+# CAR FOR YOU assets
 
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
 ```
-npm install @carforyou/example
+npm install @carforyou/assets-pkg
+```
+
+### clean `dist`
+```
+npx cfy-assets clean
+```
+
+### optimize svg from `src` to `dist`
+```
+npx cfy-assets optimize
+```
+
+### generate components from optimized svg
+```
+npx cfy-assets components
+```
+
+
+### create index
+```
+npx cfy-assets index
+```
+
+### run all commands at once without worrying about the order
+```
+npx cfy-assets build
+```
+
+## Configuration
+You can create a `assets.config.json` file in your root directory and specify the following things:
+
+| Property    | Type    | Format |
+| ----------- | ------- | ------ |
+| `rootPath`  | string  | Add the root path to your svg assets relative to your config file. Your svgs must then be placed within that root folder in an `src` directory |
+| `indexFiles`| array   | Add the path to the directory you want to create an index file of. |
+| `debug`     | boolean | If you want to log details to your console or not |
+
+**Your svgs have to be placed within a rootPath/src folder.**
+
+````json
+{
+  "rootPath": "./assets",
+  "indexFiles": [
+    {
+      "path": "bodyTypes",
+      "extension": "tsx"
+    },
+    {
+      "path": "badges",
+      "extension": "svg"
+    }
+  ],
+  "debug": false
+}
```

Comment:
interesting! I didn't think about going for a config file. how would a world look like where it's a pure CLI with arguments?

Reply:
I was thinking this is the best solution because I did not want to specify the `rootPath` for each command. But yes it is possible as a pure CLI pkg. It would look like:
`npx cfy-assets build ./assets bodyTypes tsx brands svg badges svg` instead of `npx cfy-assets build` for listing

Reply:
Can we leverage glob patterns for defining the paths instead of passing multiple? We can also run the command multiple times...

I'm open here, but before requiring another config file, I'd get some other team members opinion

Reply:
oh no my beloved config file :D had a hard time figuring this out. Requested for review from frontend :)

Reply:
I like the config file sense that it's easy to track configuration changes. What I wonder is whether:
- we can leverage conventions like assets always live in `assets/src` instead of configuration
- configuration of the tools that we're using (so put the `replaceColors` in the svgr config file in the project)

WDYT?

Reply:
- for the pkg the assets need to be placed under `project-root/src/assets` and cannot be directly under the root. But I can validate the config to only allow either `project-root/assets/src` or `project-root/src/assets/src`
- do you mean that we keep the svgo and svgr configuration completely in the consumer projects and not in this pkg?

Reply:
I'm in favour of convention over configuration, but we can enforce it any time. if this works, we should move forward. feel free to do any optimizations later